build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -88,6 +88,9 @@ featureGates:
 # Enable L7FlowExporter on Pods and Namespaces to export the application layer flows such as HTTP flows.
 {{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "L7FlowExporter" "default" false) }}
 
+# Allow users to advertise Service IPs, Pod IPs, and Egress IPs to external BGP peers.
```

Comment:
I think with Pod IPs you meant Pod CIDR here, better to change it to pod CIDR instead